lib/setupmanagers/qemuhck/qemu_machine.rb

```diff
@@ -436,11 +436,12 @@ def process_device(device_info)
     end
 
     def process_optional_hck_network
-      return unless @config['transfer_net_enabled']
+      return if !@config['transfer_net_enabled'] && !@options['share_on_host_path']
```

Comment:
It is awkward to create a transfer network when `transfer_net_enabled` is false. What about removing it?

Reply:
`transfer_net_enabled` is a config option. If for some reason we want to have a share for every run in the same folder.

Reply:
What about removing the config option?

Reply:
ok, let's remove it for now